src/App.css

```diff
@@ -1,38 +1,63 @@
-.App {
-  text-align: center;
+.container {
+  align-items: center;
+  padding: 0px;
+  background: #181818;
+}
+.topbar {
+  width: 1097px;
+  height: 119px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  display: flex;
+  flex-direction: row;
+  margin: auto;
+  justify-content: space-between;
+  align-items: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.menu {
+  align-items: center;
+  padding: 0px;
+  gap: 10px;
+  color: #fff;
+  width: 393px;
+  height: 44px;
+  display: flex;
+  justify-content: space-evenly;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.menuItem {
+  color: #FFFFFF;
+  text-decoration: none;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
+.loginButton {
+  box-sizing: border-box;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  padding: 12px 36px;
+  width: 139px;
+  height: 52px;
+  right: 39px;
```

Comment:
Use padding and adjust its size accordingly. Always use rounded figures. Either it should end in 0's or 5's.
And `right` is not necessary. Remove it.